test/v1beta1/occurrence_test.go

```diff
@@ -332,6 +335,18 @@ func TestOccurrence(t *testing.T) {
 					filter:   `resource.uri.contains("://")`,
 					expected: batchResponse.Occurrences,
 				},
+				{
+					name:   "match an occurrence with less than or equal to a certain time and greater than or equal to a certain time",
+					filter: fmt.Sprintf(`"build.provenance.createTime" <= "%[1]s" && "build.provenance.createTime" >= "%[1]s"`, buildCreateTime),
```

Comment:
This is cool. I was hoping that this would automatically support our timestamps. Nice!